poleno/utils/tests/test_forms.py

```diff
@@ -436,23 +432,6 @@ def test_invalid_values(self):
         with self.assertRaisesMessage(ValidationError, u'"" is not a valid email address'):
             validate_formatted_email(u',smith@example.com')
 
-    @unittest.skip(u'FIXME')
-    def test_normalized_values(self):
-        with self.assertRaisesMessage(ValidationError, u'Parsed as: smith@example.com'):
-            validate_formatted_email(u'<smith@example.com>')
```

Comment:
@martinmacko47 Problem nastava vo funkcii `parseaddr`, ktora sa za ten cas zmenila v spravani. Nasledovne emaily su podla validatora `validate_email` platne a nevyhadzuju chybu. Co s tym, spravime dodatocnu vlastnu validaciu, alebo to nechame tak?

Reply:
@viliambalaz Vyzera, ze povodne sme maily, ktore sa po zparsovani znormalizovali na ine stringy, nepovolovali. Zmenilo sa to v commitoch https://github.com/martinmacko47/chcemvediet/commit/00a3765668da9f602e210435ef1920a3462b06d2#diff-f51bda601eff4e117bfece46bd6eaead a https://github.com/martinmacko47/chcemvediet/commit/0460d7cc644c2b3dc490a8030aaeec517bf91383#diff-f51bda601eff4e117bfece46bd6eaead .

Takze test treba zrejme prerobit na pozitivny. Tj. ze `validate_formatted_email` na dane pripady povoli. Ku kazdomu riadku mozes pridat do komentara `// Parsed as: invalidsmith@example.com` ako bola povodna error hlaska.

Reply:
Nepouzil som presne znenie error hlasky. Pouzil som hodnoty z `name, address = parseaddr(value)`. Dva emaily som oddelil `;`.

Reply:
@viliambalaz Do tych komentarov davaj iba `address`, tak ako je v error hlaske. Lebo string "John smith@example.com" je absolutna nevalidna blbost. Proste funkcia `test_normalized_values` validuje mail. Tak do poznamky sa zide dat, co z toho sa berie ako mail. "John" v tom priklade nie je sucast mailu.

Reply:
@viliambalaz Pls tam tam presne ako to bolo v tych error hlaskach. Lebo nieco ako `John smith@example.com` je totalny nezmysel. Spravne je `John <smith@example.com>`.

Reply:
@viliambalaz Pls oddel testy mailov, ktore su sparsovane 1 k 1 od testov mailov, ktore su znormalizovane. Idealne tak ako to bolo. `test_valid_values` a `test_normalized_values`. Nechapem naco si to spajal. Teraz je to iba neprehladne.